components/Trade/Advanced/RightPanel/InsuranceInfo/index.tsx

```diff
@@ -24,7 +26,7 @@ export default styled(({ className }) => {
                     },
                 }}
             >
-                {toApproxCurrency(poolInfo?.liquidity ?? defaults.liquidity)}
+                {toApproxCurrency(parseFloat(poolLiquidity.toFixed(2)) ?? defaults.liquidity)}
```

Comment:
you dont need to round for toApproxCurrency, just look at the implementation of toApprox currency and change the default to 2

Reply:
which you have already done so you can revert these ones

Reply:
Rectified